pkg/vault/status.go

```diff
@@ -5,11 +5,11 @@ import (
 	"time"
 )
 
-func (v *Vault) isVaultHealthy() (bool, error) {
+func (v *Vault) isVaultHealthy() (bool, Error) {
 
 	result, err := v.client.Sys().Health()
 	if err != nil {
-		return false, v.parseError(err)
+		return false, v.parseError(err).(Error)
```

Comment:
Do you really have to cast this?  I would not think so.

Reply:
fixed